ensime-mode.el

```diff
@@ -472,13 +472,14 @@
 
 (defun ensime-troubleshooting (arg)
   "Information about emacs and ensime installation.
-A prefix argument will add the report to the kill ring."
+A prefix argument will prevent to add the report to the kill ring."
```

Comment:
this is overengineering, no need to handle an alt case, let's just add to the kill ring.